include/sdbus-c++/Message.h

```diff
@@ -195,9 +196,13 @@ namespace sdbus {
         void dontExpectReply();
         bool doesntExpectReply() const;
 
+    protected:
+        MethodCall(void *msg, internal::ISdBus* sdbus, internal::IConnection* connection, adopt_message_t) noexcept;
+
     private:
         MethodReply sendWithReply(uint64_t timeout = 0) const;
         MethodReply sendWithNoReply() const;
+        internal::IConnection* connection_{};
```

Comment:
I would assign a `nullptr` here to be more explicit.

Reply:
I kept the style which is already in use, also in the same file. E.g: https://github.com/Kistler-Group/sdbus-cpp/blob/ef6b5ea1384172bf0266a06e1fdea58e07bd3941/include/sdbus-c%2B%2B/Message.h#L172